bindgen/ir/var.rs

```diff
@@ -389,9 +392,94 @@ impl ClangSubItemParser for Var {
     }
 }
 
+fn parse_macro_clang_fallback(
+    ctx: &mut BindgenContext,
+    cursor: &clang::Cursor,
+) -> Option<(Vec<u8>, cexpr::expr::EvalResult)> {
+    let file = ".macro_eval.c".to_string();
```

Comment:
why to_string? This is a string literal and you're not mutating it so seems just redundant / unnecessary overhead.